Doc/includes/sqlite3/execute_1.py

```diff
@@ -10,9 +10,17 @@
 # This is the qmark style:
 cur.execute("insert into people values (?, ?)", (who, age))
 
-# And this is the named style:
-cur.execute("select * from people where name_last=:who and age=:age", {"who": who, "age": age})
+# The qmark style used with executemany():
+people = [
+    ('Chirac', 70),
+    ('Finnbogadóttir', 72),
+    ('Schröder', 58),
+]
+cur.executemany("insert into people values (?, ?)", people)
 
-print(cur.fetchone())
+# And this is the named style:
+cur.execute("select * from people where name_last=:who and age=:age",
+            {"who": who, "age": age})
```

Comment:
The executemany() example will make people to look up where did who and age come from. I think it would be better to get rid of them and pass name and age directly.

Reply:
And while we are here, let's get rid of Yeltsin and others and use non-politicians as an example.